koku/api/settings/utils.py

```diff
@@ -82,3 +88,54 @@ def create_dual_list_select(name, left_options=[], right_options=[], **kwargs):
     dual_list_select = {"component": "dual-list-select", "name": name}
     dual_list_select.update(**kwargs)
     return dual_list_select
+
+
+def create_select(name, **kwargs):
+    """
+    Create a select for the settings.
+
+    Args:
+        (String) name - unique name for switch.
+
+    Returns:
+        [Dict] - Subform component.
+    """
+    select = {"component": "select", "name": name}
+    select.update(**kwargs)
+    return select
+
+
+"""Common utilities and helpers for Currency."""
+
+
+def get_selected_currency_or_setup(schema):
```

Comment:
chore: let's add docstrings to these functions.

Reply:
Done